Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -243,10 +285,14 @@ final class GoogleSheetsManager: ObservableObject {
   
   func addTransaction(amount: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int, completion: @escaping (Bool) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/25c2f5d2b54fb8275ada47af8d5a461ce85fbfd7/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L286"></span>Function should have 5 parameters or less: it currently has 7<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:286</code></td>
    </tr>
  </tbody>
</table>